src/server/server.cc

```diff
@@ -1839,6 +1844,26 @@ void Server::cleanupExitedWorkerThreads(bool force) {
   }
 }
 
+Status Server::forceClusterMode() {
+  std::string value;
+  auto cf = storage->GetCFHandle(engine::kPropagateColumnFamilyName);
+  rocksdb::Status check_cluster_enabled =
+      storage->Get(rocksdb::ReadOptions(), cf, rocksdb::Slice(engine::kClusterEnabledKey), &value);
+
+  if (check_cluster_enabled.IsNotFound()) {
+    Status s = storage->WriteToPropagateCF(engine::kClusterEnabledKey, std::to_string(config_->cluster_enabled));
+    if (!s.IsOK()) return s;
```

Comment:
Can return the status directly if the key is not found
```suggestion
    return storage->WriteToPropagateCF(engine::kClusterEnabledKey, std::to_string(config_->cluster_enabled));
```